lego/apps/companies/serializers.py

```diff
@@ -79,10 +80,21 @@ class CompanyListSerializer(BasisModelSerializer):
         }
     )
 
+    event_count = serializers.SerializerMethodField()
+
+    def get_event_count(self, obj):
```

Comment:
We generally have all the class values before functions in our serielizer, so the line `joblisting_count = serializers.SerializerMethodField()` should be over this function :smile:

Reply:
got it! Will fix